masonry/src/widgets/button.rs

```diff
@@ -79,7 +79,7 @@ impl Button {
         Label::set_text(&mut Self::label_mut(this), new_text);
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Get a mutable reference to the label.
```

Comment:
This is *true*, but you also get the `MutateCtx` along for the ride. I'm not sure what the right way to say this succinctly is.

Reply:
Honestly, for some of these items the signature is self-explanatory; the documentation only serves to say "Yes, this does what the signature implies".